wiki/BanchoBot/fr.md

```diff
@@ -135,7 +135,7 @@ Il y a un total de 7 statuts qui peuvent être affichés par la commande `!stats
 
 ```
 13:04 pippi: !faq fr:peppy
-13:04 BanchoBot: Qui ne le connaît pas ? peppy est le créateur de osu!
```

Comment:
FWIW it's a direct quote from `!faq fr:peppy`, but I guess that could be fixed

Reply:
very cool, how can i contribute to that

Reply:
I'll ask someone from the French GMs to proofread FAQs

Reply:
alright, tell them to also relay updates to this article or one of us